src/Entity/Field/MarkdownField.php

```diff
@@ -20,4 +21,18 @@ public function __toString(): string
 
         return $markdown->toHtml(reset($value));
     }
+
+    /**
+     * @return string|array|Markup
+     */
+    public function getTwigValue()
+    {
+        $value = (string) $this;
```

Comment:
How this one differs from the base Field class?

Reply:
By calling `(string) $this` it gets parsed as Markdown. Otherwise it would return the "stored" value, unparsed.

Reply:
same should happen inside getFlattenedValue as it is used in API response
(maybe rename to getParsedValue?)

Reply:
Hmm, makes sense.. I'll work on this some more.